server/src/main/java/org/opensearch/index/codec/customcodecs/Lucene95CustomCodec.java

```diff
@@ -8,19 +8,27 @@
 
 package org.opensearch.index.codec.customcodecs;
 
-import org.apache.lucene.codecs.StoredFieldsFormat;
+import org.apache.logging.log4j.Logger;
 import org.apache.lucene.codecs.FilterCodec;
+import org.apache.lucene.codecs.StoredFieldsFormat;
 import org.apache.lucene.codecs.lucene95.Lucene95Codec;
+import org.opensearch.index.codec.PerFieldMappingPostingFormatCodec;
+import org.opensearch.index.mapper.MapperService;
 
-import java.util.Locale;
-
-abstract class Lucene95CustomCodec extends FilterCodec {
+/**
+ *
+ * Extends {@link FilterCodec} to reuse the functionality of Lucene Codec.
+ * Supports two modes zstd and zstd_no_dict.
+ *
+ * @opensearch.internal
+ */
+public abstract class Lucene95CustomCodec extends FilterCodec {
     public static final int DEFAULT_COMPRESSION_LEVEL = 6;
 
     /** Each mode represents a compression algorithm. */
     public enum Mode {
         ZSTD,
-        ZSTDNODICT
+        ZSTD_NO_DICT
```

Comment:
The rename part I get, what if some users already installed the sandbox plugin? Migration would fail, right?

Reply:
yes, it would fail.

Reply:
I find naming convention being compelling enough :) `ZSTD_NO_DICT` "seems" better?

Reply:
Sure, no objections (I mean we let `ZSTDNODICT` in, so it looked fine at that time :-) but could be certainly overlooked), @sarthakaggarwal97 could you please create a follow up issue to update https://github.com/opensearch-project/opensearch-benchmark?

Reply:
@reta @dblock folks, have opened the [issue](https://github.com/opensearch-project/opensearch-benchmark/issues/335) with OSB